pymongo/event_listeners.py

```diff
@@ -0,0 +1,130 @@
+import logging
+
+from pymongo import monitoring
+
+class CommandLogger(monitoring.CommandListener):
```

Comment:
We should stick to calling these `Loggers` or `Listeners`. Currently, the submodule is called `event_listener` while the classes are called `*Logger` which is poor UX.